synnax/pkg/api/frame_streamer.go

```diff
@@ -51,7 +51,7 @@ func (s *FrameService) Stream(ctx context.Context, stream StreamerStream) error
 	plumber.SetSource[FrameStreamerRequest](pipe, "receiver", receiver)
 	plumber.MustConnect[FrameStreamerResponse](pipe, "streamer", "sender", 70)
 	plumber.MustConnect[FrameStreamerRequest](pipe, "receiver", "streamer", 70)
-	pipe.Flow(sCtx, confluence.CloseInletsOnExit())
+	pipe.Flow(sCtx, confluence.CloseInletsOnExit(), confluence.CancelOnFail())
```

Comment:
it might be worthwhile to add comments in options.go explaining the options because CancelOnFail() isn't immediately obvious as to what it enables (or what is getting cancelled)/ when I should and shouldnt use it.

Reply:
Good idea.

Reply:
Leo is doing this in his PR

Reply:
@LeonLiur

Reply:
Yes